docs/cli/commands/kcp_taskrun.md

```diff
@@ -0,0 +1,69 @@
+---
+title: kcp taskrun
+---
+Run generic tasks on one or more Kyma runtimes
+
+## Synopsis
+
+Runs a command (which can be a script or a program with arbitrary arguments) on targets of Kyma runtimes.
+The specified command will be executed locally in parallel in separate subprocesses for each runtime, where the number of parallel executions are controlled by the --parallelism option.
+
+For each subprocess, the following runtime specific data are passed as environment variables:
+  KUBECONFIG       : Path to the kubeconfig file for the specific runtimme
+  GLOBALACCOUNT_ID : Global Account ID of the runtime
+  SUBACCOUNT_ID    : Subaccount ID of the runtime
+  RUNTIME_NAME     : Shoot cluster name
+  RUNTIME_ID       : Runtime ID of the runtime
+
+The exit status is zero (0) if all subprocesses exit successfully with zero status code. If one or more subprocesses exit with non-zero status, the command will also exit with non-zero status.
+
+```bash
+kcp taskrun --target <TARGET SPEC> ... [--target-exclude <TARGET SPEC> ...] COMMAND [ARGS ...] [flags]
+```
+
+## Examples
+
+```
+  kcp taskrun --target all kubectl patch deployment valid-deployment -p '{"spec":{"template":{"spec":{"containers":[{"name":"kubernetes-serve-hostname","image":"new image"}]}}}}'
+    Execute a kubectl patch operation for all runtimes
+  kcp taskrun --target account=CA4836781TID000000000123456789 /usr/local/bin/awesome-script.sh
+    Run a maintenance script for all runtimes of a given Global Account
+  kcp taskrun --target all helm upgrade -i -n kyma-system my-kyma-addon --values overrides.yaml
+    Deploy or a helm chart on all runtimes
+```
+
+## Options
+
+```
+      --keep                         Keep downloaded kubeconfigs after execution for caching purpose
+      --kubeconfig-dir string        Directory to download runtime kubeconfigs to. By default it is a random-generated directory in the OS specific default temporary directory (e.g. /tmp in Linux)
+  -p, --parallelism int              Number of parallel commands to execute (default 8)
```

Comment:
```suggestion
  -p, --parallelism int              Number of parallel commands to execute. The default value is 8.
```

Reply:
default is auto-generated, cannot rephrase.